lib/union/union.test.js

```diff
@@ -1,8 +1,8 @@
 var union = require('.')
 
 test('returns concatenated array of two with only unique values', () => {
-	const func = function() { return 'a' }
-	const func2 = function() { return 'a' }
+	const func = function() {  return 'a' }
+	const func2 = function() { return 'b' }
```

Comment:
If both functions will be the same but have different references, test will be failed

```js
const func1 = function() { return 'a' }
const func2 = function() { return 'a' }
```

Reply:
yeah, we should consider this
same for two objects